libs/orders/src/lib/components/order-feedback/order-feedback.tsx

```diff
@@ -113,6 +74,27 @@ export const OrderFeedback = ({ transaction, order }: OrderFeedbackProps) => {
           </div>
         )}
       </div>
+      {orderRejectionReason && (
+        <div>
+          <p className={labelClass}>{t(`Reason`)}</p>
+          <p data-testid="error-reason">{t(orderRejectionReason)}</p>
+        </div>
+      )}
     </div>
   );
 };
+
+const getRejectionReason = (
+  order: OrderEvent_busEvents_event_Order
+): string | null => {
+  switch (order.status) {
+    case OrderStatus.Stopped:
+      return t(
+        `Your ${order.timeInForce} order was not filled and it has been stopped`
```

Comment:
This will be revisited as soon as we get a `reason` behind why the order has been stopped by Vega. @campbellssource